roles/container_runtime/templates/crio.conf.j2

```diff
@@ -141,16 +141,22 @@ signature_policy = ""
 # The valid values are mkdir and ignore.
 image_volumes = "mkdir"
 
+# CRI-O reads its configured registries defaults from the containers/image configuration
+# file, /etc/containers/registries.conf. Modify registries.conf if you want to
+# change default registries for all tools that use containers/image.  If you
+# want to modify just crio, you can change the registies configuration in this
+# file.
+
 # insecure_registries is used to skip TLS verification when pulling images.
-insecure_registries = [
-{{ l_insecure_crio_registries|default("") }}
-]
+# insecure_registries = [
+# {{ l_insecure_crio_registries|default("") }}
```

Comment:
We should make sure that the registries set by this variable are populated in /etc/containers/registries.conf.

Reply:
It is being populated in /etc/containers/registries.conf - https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/templates/registries.conf#L31.

Reply:
Yeah, but I don't see l_insecure_crio_registries. If any users are using that variable, this change would break them.

Reply:
Right, I didn't see that. Fixed now.

Reply:
There's no sense in leaving these commented variables in place.  Let's remove them, also remove them from defaults/main.yml since it appears they were only used here.

Change to 
\# insecure_registries = []